src/vm/gdbjit.cpp

```diff
@@ -554,6 +584,15 @@ GetDebugInfoFromPDB(MethodDesc* MethodDescPtr, SymbolsInfo** symInfo, unsigned i
         }
     }
 
+    if (methodDebugInfo.locals)
```

Comment:
It would be better to add a destructor to the MethodDebugInfo and let it destroy the locals and points. Then just letting the methodDebugInfo go out of scope would perform the proper cleanup. and it would enable proper cleanup even in case of exeption unwinding. I have suggested to convert all nothrow news in this file to throwing ones in #10033 and with that, automatic cleanup would be required.